### PR TITLE
[IEFRAME] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Improvements based on recent Romanian translation document revision 06e89b2.

This files was not translated in Romanian language in any Win version (neither XP/2k3+, nor in the previous ones). I tried to translate this file as close as possible in the way I did in the previous PRs.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian for review

File: dll/win32/ieframe/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /*
- * PROJECT:     ReactOS ieframe.dll
+ * PROJECT:     ReactOS Internet Explorer main frame window
```
Comment (binarymaster) [inline dll/win32/ieframe/lang/ro-RO.rc:2]:
```suggestion
 * PROJECT:     ReactOS ieframe.dll
```

**Comment by AndreiMiloiu** [comment]
Thank you @binarymaster!

File: dll/win32/ieframe/lang/ro-RO.rc [hunk 2]
```diff
@@ -66,24 +66,24 @@ STRINGTABLE
...
 IDD_BROWSE_OPEN DIALOG 10, 10, 200, 70
 STYLE DS_MODALFRAME | WS_CAPTION
-CAPTION "Deschide URL"
+CAPTION "Deschidere URL"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "Specificați adresa URL pe care doriți s-o deschideți în Internet Explorer",-1, 35,5,160,25
-    LTEXT "Deschide:", -1, 5, 32, 30, 15
+    LTEXT "Specificați adresa URL pe care doriți să o deschideți în Internet Explorer",-1, 35,5,160,25
+    LTEXT "Deschidere:", -1, 5, 32, 30, 15
```
Comment (JoachimHenze) [inline dll/win32/ieframe/lang/ro-RO.rc:82]: Prove with a pic that the longer string "Deschidere:" would actually fit the 30 width. I do have my doubt on that.

Comment (AndreiMiloiu) [reply dll/win32/ieframe/lang/ro-RO.rc:82]: I will revert this change. It doesn't fit in the box. 

2024-09-25 21_22_15-Dialog - 1001

**Pull request merged**